pkg/stores/sqlproxy/proxy_store.go

```diff
@@ -749,8 +752,10 @@ func (s *Store) ListByPartitions(apiOp *types.APIRequest, schema *types.APISchem
 	fields := getFieldsFromSchema(schema)
 	fields = append(fields, getFieldForGVK(gvk)...)
 	transformFunc := s.transformBuilder.GetTransformFunc(gvk)
-
-	inf, err := s.cacheFactory.CacheFor(fields, transformFunc, &tablelistconvert.Client{ResourceInterface: client}, attributes.GVK(schema), attributes.Namespaced(schema), controllerschema.IsListWatchable(schema))
+	client2 := &tablelistconvert.Client{ResourceInterface: client}
+	attrs2 := attributes.GVK(schema)
+	ns2 := attributes.Namespaced(schema)
```

Comment:
I didn't see a second attributes or namespace in that function so we can remove the `2` suffix.

```suggestion
	tableClient := &tablelistconvert.Client{ResourceInterface: client}
	attrs := attributes.GVK(schema)
	ns := attributes.Namespaced(schema)
```

Reply:
I broke these out from the long call to `s.cacheFactory.CacheFor` because the overly long line was harder to digest, and frustrating when stepping through this code in the debugger.